raiden/transfer/utils.py

```diff
@@ -125,7 +125,7 @@ def hash_balance_data(
     assert locksroot != b''
     assert len(locksroot) == 32
     if transferred_amount == 0 and locked_amount == 0 and locksroot == EMPTY_HASH:
-        return EMPTY_HASH
+        return BalanceHash(EMPTY_HASH)
```

Comment:
This should be typed constant in `constant.py`

Reply:
Good point.